Makefile

```diff
@@ -92,7 +92,7 @@ EXTRA_CLEAN = $(addprefix $(ag_regress_dir)/, $(ag_regress_out)) src/backend/par
 ag_include_dir = $(srcdir)/src/include
 PG_CPPFLAGS = -I$(ag_include_dir) -I$(ag_include_dir)/parser
 
-PG_CONFIG = pg_config
+PG_CONFIG ?= pg_config
```

Comment:
Previously, it always assigning a new variable. so, It can't assigning from Shell.

like ```PG_CONFIG=~/pg11/pg_config make -j16```